src/azul/service/elasticsearch_service.py

```diff
@@ -537,21 +539,80 @@ def transform_summary(self,
                 'sum',
                 field='contents.cell_suspensions.total_estimated_cells_'
             )
-            # Add a total cell count aggregate
+            # FIXME: Remove deprecated field totalCellCount
+            #        https://github.com/DataBiosphere/azul/issues/3650
+            # Add a cell suspensions cell count aggregate
             es_search.aggs.metric('totalCellCount',
                                   'sum',
                                   field='contents.cell_suspensions.total_estimated_cells_')
+            # Add a cell suspensions cell count aggregate from projects
+            # that have no project level estimated cell count.
+            field_full = translation['projectEstimatedCellCount']
+            field_type = self.field_type(catalog, tuple(field_full.split('.')))
+            null_value = field_type.to_index(None)
+            es_search.aggs.bucket(
+                'withoutProjectCellCount',
+                'filter',
+                filter=Q('terms', **{field_full: [0, null_value]})
+            ).bucket(
+                'totalCellCount',
+                'sum',
+                field='contents.cell_suspensions.total_estimated_cells_'
```

Comment:
but then hard-code the shadow field name. It's still brittle this way. Either remove the lookup or derive this.